### ISSUE TITLE
POV icon doesn't follow UI theme

### ISSUE THREAD

**Issue by Quitch** [Legion-Expansion/Legion-Expansion#572]
The latest PA build adds a POV button to the bottom bar and to the PIP. These should be coloured to adhere to the Legion UI theme.

**Comment by Quitch** [comment]
While an icon was submitted, it was never tested and has never worked. The issue appears to be that the pov.png file is also referenced in live_game_pip.css, which is referenced by live_game_pip_tl.html. It would seem that we'd need to setup something such that when the Legion theme is in use, we load our own CSS to ensure our button is loaded.

**Comment by Quitch** [comment]
For some reason the code:

$('img[src="img/ingame_options_bar/pov.png]').attr("src","coui://ui/mods/com.pa.legion-expansion/img/ingame_options_bar/red/pov.png");

is returning []. This is the same mechanism used to replace all the other buttons.

**Comment by Quitch** [comment]
@wondible I've been looking into why the Legion POV button isn't being used and I can't find anything wrong. In client\ui\mods\com.pa.legion-expansion\live_game_options_bar.js we have lines that look as follows:
                $('img[src="img/ingame_options_bar/chronocam.png"]').attr("src","coui://ui/mods/com.pa.legion-expansion/img/ingame_options_bar/red/chronocam.png");
                $('img[src="img/ingame_options_bar/game_menu.png"]').attr("src","coui://ui/mods/com.pa.legion-expansion/img/ingame_options_bar/red/game_menu.png");
                $('img[src="img/ingame_options_bar/pov.png]').attr("src","coui://ui/mods/com.pa.legion-expansion/img/ingame_options_bar/red/pov.png");

I've validated the paths, both Legion and source. Everything is fine. The lines which change the image for all the other buttons on the options bar work fine. The Legion POV button appears in-game if you replace a different button with it. Yet the current POV button will not change. Running the line in console produces:

$('img[src="img/ingame_options_bar/pov.png]').attr("src","coui://ui/mods/com.pa.legion-expansion/img/ingame_options_bar/red/pov.png");
[]

Yet examining the scene's elements I can see that img/ingame_options_bar/pov.png is indeed the source.

About the only thing I have uncovered is that in the game's live_game_options_bar.html, the POV button is using some duplicate class and id stuff from the time controls button that looks iffy (which I have reported to mikey for fixing). On the off-chance it was involved I removed "div_time_invoke_cont" from the class and replaced "btn_time_controls" with a unique ID, but this had no effect.

I'm at a bit of a loss now as everything looks OK in the code to me, so I'm not sure why [] is being returned
